includes/html/functions.inc.php

```diff
@@ -963,7 +963,10 @@ function alert_details($details)
         }
 
         if ($tmp_alerts['sensor_id']) {
-            $details = 'Current Value: ' . $tmp_alerts['sensor_current'] . ' (' . $tmp_alerts['sensor_class'] . ')<br>  ';
+            $details = 'Value: ' . $tmp_alerts['sensor_current'] . ' (' . $tmp_alerts['sensor_class'] . ')<br>  ';
```

Comment:
why not this in a else part of the if clause ?

Reply:
For a while, the style check did not like else statements. Don't know if it now stopped complaining about this. But I avoid else statements when easily avoidable. 
But both are completely equivalent for sure.

Reply:
those check is not more included :-)
"else" is/was fine everytime, even if a style checker was against it ;-)

Reply:
True, but that variant is also fine :) got used to it :) So unless you want me to remove it ... 

To solve this case in a proper way, I still have in my todo list to use blades and define this piece of text in the alert list directly from them. Each rule would then be able to adapt how it is displayed in the list, with all but only the necessary details.  But not today :)

Reply:
yeah changing stuff to blade is a big task :-)
nethertheless i would recommend it to have it this way:

if ($tmp_alerts['sensor_class'] == 'state') 
   detail = ...
else
   detail =


change it and you'll get my approve
deal? ;-)

Reply:
That's a negotiation... 
But agreed :D 

Was lazy to do a `git checkout` to change :)